docs/installation.md

```diff
@@ -143,6 +154,8 @@ server {
         fastcgi_pass unix:/run/php/php8.1-fpm.sock;
         fastcgi_index index.php;
         fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
+######## You may need change #document_root to your public folder if you are running a FreeBSD system.
+	# fastcgi_param SCRIPT_FILENAME /var/www/Lychee/public$fastcgi_script_name;
```

Comment:
What we set here appears to be the default, so I'm not sure why we're setting it at all. Your comment feels confusing in its current form - will it not accept `$document_root`?

```suggestion
######## You may need to replace $document_root with the absolute path to your public folder.
        fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
```

Reply:
Acutally I don't know if it's only applying for FreeBSD. But if I don't change $document_root to absolute path, Nginx will return "File not found".
I think maybe relative path has some bugs on FreeBSD. Because I also met a ffmpeg probelm, video donot have thumbs and has_ffmpeg, has_exiftools are both set to 0. I fix it by add absolute path of FFprobe in ../vendor/lychee-org/php-exif/lib/PHPExif/Adapter/FFprobe.php according to https://lycheeorg.github.io/docs/faq.html#why-dont-my-videos-have-thumbnails.
> protected $toolPath = '/usr/local/bin/ffprobe';

In addition, I don't have Lychee/app/ModelFunctions/PhotoFunctions.php (prior to 4.2.0) or Lychee/app/Actions/Photo/Extensions/VideoEditing.php (starting with 4.2.0), but ffmpeg works well after I fixed FFprobe path issue. Maybe from which new version, Lychee removed or changed it?